recipes/pynndescent/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "pynndescent" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b58832a8f56d6d89018c5d2e9fce866ec7a5fa7b1a5aa02363d6ef2015929b31
+
+build:
+  number: 0
+  skip: True  # [py27 or win32]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+  host:
+    - python
```

Comment:
```suggestion
    - python >=3.6
```

Reply:
Ah, thanks -- wasn't sure the best way to do this.